src/libraries/System.Net.Security/src/System/Net/CertificateValidationPal.Windows.cs

```diff
@@ -89,6 +90,43 @@ internal static partial class CertificateValidationPal
             return result;
         }
 
+        // Check that local certificate was used by schannel.
+        internal static bool IsLocalCertificateUsed(SafeFreeCredentials? _credentialsHandle, SafeDeleteContext securityContext)
```

Comment:
Why is this parameter prefixed with an underscore?  It makes all access sites look like they're referencing a field.